src/gwt/src/org/rstudio/studio/client/workbench/views/output/find/FindInFilesDialog.java

```diff
@@ -203,12 +225,26 @@ protected State collectInput()
             list.add(trimmedPattern);
       }
 
+      String excludeFilePatterns =
+         listPresetExcludeFilePatterns_.getValue(
+               listPresetExcludeFilePatterns_.getSelectedIndex());
+      if (excludeFilePatterns == "custom")
```

Comment:
Nit: for very complicated reasons, we prefer StringUtil.equals (GWT has actually changed the way string comparison works between releases!)

Reply:
Fixed